### PR TITLE
[WIP] Adding kraken to Scale-CI pipeline

### PR DESCRIPTION
**Pull request by yashashreesuresh**
This commit:

- Adds a kraken template which will be managed by scale-ci-watcher.
  Kraken workload allows us to inject chaos scenarios into the cluster. 
  Thereby helps in determining cluster's resilience.

- Adds a sample properties file to kick off the Job.

### CODE REVIEW

**Comment by yashashreesuresh** [comment]
Please have a look @chaitanyaenr

File: jjb/dynamic/scale-ci_kraken.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+        time ansible-playbook -vv -i inventory kraken.yml
+    concurrent: true
+    description: This job runs Kraken workload
+    disabled: false
+    name: kraken_jjb
+    node: dev-scripts&&test_benchmark
+    parameters:
+    - string:
+        default: "ec2-34-220-197-160.us-west-2.compute.amazonaws.com"
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_kraken.yml:27]: we need to remove the default.

File: jjb/dynamic/scale-ci_kraken.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        # Create inventory File:
+        echo "[orchestration]" > inventory
+        echo "${ORCHESTRATION_HOST}" >> inventory
+
+        time ansible-playbook -vv -i inventory kraken.yml
+    concurrent: true
+    description: This job runs Kraken workload
+    disabled: false
+    name: kraken_jjb
+    node: dev-scripts&&test_benchmark
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_kraken.yml:24]: s/dev-scripts&&test_benchmark/scale-ci

File: pipeline-scripts/kraken.groovy [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            def scenarios = baseline_properties['SCENARIOS']
+            def cerberus_enabled = baseline_properties['CERBERUS_ENABLED']
+            def cerberus_url = baseline_properties['CERBERUS_URL']
+            def wait_duration = baseline_properties['WAIT_DURATION']
+            def iterations = baseline_properties['ITERATIONS']
+            def daemon_mode = baseline_properties['DAEMON_MODE']
+
+			// Run kraken job
+			try {
+				baseline_build = build job: 'BASELINE-SCALE-TEST',
```
Comment (chaitanyaenr) [inline pipeline-scripts/kraken.groovy:41]: s/baseline/kraken

File: pipeline-scripts/kraken.groovy [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+				println "Looks like the property file already exists, erasing it"
+				sh "rm ${property_file_name}"
+			}
+			// get properties file
+			sh "wget ${BASELINE_PROPERTY_FILE} -O ${property_file_name}"
+			sh "cat ${property_file_name}"
+			def baseline_properties = readProperties file: property_file_name
+			def orchestration_host = baseline_properties['ORCHESTRATION_HOST']
+			def orchestration_user = baseline_properties['ORCHESTRATION_USER']
+            def sshkey_token = baseline_properties['SSHKEY_TOKEN']
```
Comment (chaitanyaenr) [inline pipeline-scripts/kraken.groovy:26]: tabs vs spaces issue

File: jjb/dynamic/scale-ci_kraken.yml [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+        trim: 'false'
+    - string:
+        default: "/root/.kube/config"
+        description: Location of kubeconfig on orchestration host.
+        name: KUBECONFIG_PATH
+        trim: 'false'
+    - string:
+        default: "/root/kraken"
+        description: Kraken folder location on jump host.
+        name: KRAKEN_FOLDER
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_kraken.yml:49]: s/FOLDER/DIR

File: jjb/dynamic/scale-ci_kraken.yml [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+        # Create inventory File:
+        echo "[orchestration]" > inventory
+        echo "${ORCHESTRATION_HOST}" >> inventory
+
+        time ansible-playbook -vv -i inventory kraken.yml
+    concurrent: true
+    description: This job runs Kraken workload
+    disabled: false
+    name: kraken_jjb
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_kraken.yml:23]: s/kraken_jjb/kraken

File: pipeline-scripts/kraken.groovy [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+			def exit_on_failure = kraken_properties['EXIT_ON_FAILURE']
+			def cerberus_enabled = kraken_properties['CERBERUS_ENABLED']
+			def cerberus_url = kraken_properties['CERBERUS_URL']
+			def wait_duration = kraken_properties['WAIT_DURATION']
+			def iterations = kraken_properties['ITERATIONS']
+			def daemon_mode = kraken_properties['DAEMON_MODE']
+
+			// Run kraken job
+			try {
+				kraken_build = build job: 'kraken_jjb',
```
Comment (chaitanyaenr) [inline pipeline-scripts/kraken.groovy:42]: s/kraken_jjb/kraken

File: jjb/dynamic/scale-ci_kraken.yml (hunk 6 above)
Comment (yashashreesuresh) [reply jjb/dynamic/scale-ci_kraken.yml]: Changed it.

File: pipeline-scripts/kraken.groovy (hunk 7 above)
Comment (yashashreesuresh) [reply pipeline-scripts/kraken.groovy]: Changed it.

**Pull request merged**